src/modules/fancyzones/FancyZonesLib/trace.cpp

```diff
@@ -14,7 +14,7 @@
 #define EventEditorLaunchKey "FancyZones_EditorLaunch"
 #define EventSettingsKey "FancyZones_Settings"
 #define EventDesktopChangedKey "FancyZones_VirtualDesktopChanged"
-#define EventZoneWindowKeyUpKey "FancyZones_ZoneWindowKeyUp"
+#define EventWorkAreaKeyUpKey "FancyZones_WorkAreaKeyUp"
```

Comment:
@jaimecbernardo is it ok to rename trace keys?

Reply:
Should I revert it to be on the safe side?

Reply:
Please revert those, indeed.
The main reason is that it has to be coordinated with analytics team.
These might end up being used currently.
There's a broader item to review the names, but let's not do that as part of this PR.

Reply:
Reverted (back to `"FancyZones_ZoneWindowKeyUp"`).